internal/apps/dop/providers/projectpipeline/service_impl.go

```diff
@@ -480,6 +481,20 @@ func (p *ProjectPipelineService) createCronIfNotExist(definition *dpb.PipelineDe
 		return err
 	}
 
+	createV2 := extra.CreateRequest
```

Comment:
what's this

Reply:
the conflict was not resolved properly and has been resolved